src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -44,6 +49,14 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
             .transform))
   }
 
+  private val subtreeCogniteIds: Option[List[CogniteId]] =
```

Comment:
I see that we're doing this for sequence rows, but I think it's a bit terrible. I'd much rather have us be explicit as we are with "spaceExternalId" etc.